packs/data/feat-effects.db/stance-crane-stance.json

```diff
@@ -60,11 +60,9 @@
                 "title": "Crane Stance"
             },
             {
-                "key": "ActiveEffectLike",
-                "mode": "override",
-                "path": "flags.pf2e.rollOptions.all.crane-stance",
-                "priority": 10,
-                "value": true
+                "domain": "all",
```

Comment:
This RollOption may not be needed any longer either.

Reply:
It is still needed.